sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -119,6 +119,18 @@ def test_wait_get():
     assert len(res) == 1
 
 
+@pytest.mark.remote_data
+def test_wait_get_as_is():
+    responses = client.search(
+        a.Time('2012/1/1T1:00:36', '2012/1/1T01:00:38'),
+        a.jsoc.Series('hmi.M_45s'), a.jsoc.Notify('jsoc@cadair.com'),
```

Comment:
Is @Cadair okay with having this email address here?

Reply:
I think we have a test email address for sunpy somewhere in this file.

Reply:
Everywhere in the file, the same email is present.
example: https://github.com/sunpy/sunpy/blob/11a0d65fc7ecfe9d85210603521e67a02cf5a7fe/sunpy/net/jsoc/tests/test_jsoc.py#L56
https://github.com/sunpy/sunpy/blob/11a0d65fc7ecfe9d85210603521e67a02cf5a7fe/sunpy/net/jsoc/tests/test_jsoc.py#L67

Reply:
Ah that is the test email. I thought there was a sunpy@sunpy one.

Reply:
Now I have the ability to create `@sunpy.org` emails, I should probably put changing this on my todo. I did, however, create this email alias for this purpose (and then deleted it as soon as I had completed the registration).